hack/csvinflater/csvinflater.go

```diff
@@ -0,0 +1,165 @@
+package main
```

Comment:
Document what this is for. Especially because "CSV" has a different meaning in most of the computing world...

Reply:
Good point.